src/Mantasflowers.Domain/Entities/OrderContactInfo.cs

```diff
@@ -0,0 +1,9 @@
+namespace Mantasflowers.Domain.Entities
+{
+    public class OrderContactInfo : BaseEntity
```

Comment:
Common fields should be moved to an abstract class ContactInfo,
```cs
public abstract class ContactInfo : BaseEntity
{
    public string Email { get; set; }

    public string Phone { get; set; }
}
```
whereas:
```cs
public class OrderContactInfo : ContactInfo
```
and 
```cs
public class UserContactInfo : ContactInfo
```
could just inherit it.